<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CreatingProjectBasedOnExistingDialog</class>
 <widget class="QDialog" name="CreatingProjectBasedOnExistingDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>469</width>
    <height>540</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <widget class="QGroupBox" name="p_groupBox_ProjectName">
     <property name="title">
      <string>Enter a new project name</string>
     </property>
     <layout class="QFormLayout" name="formLayout_2">
      <item row="0" column="0">
       <widget class="QLabel" name="p_lbl_Name">
        <property name="text">
         <string>Name:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="p_line_ProjectName"/>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="p_groupBox_PeriodTime">
     <property name="title">
      <string>New period time</string>
     </property>
     <layout class="QFormLayout" name="formLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="p_lbl_StartPeriod">
        <property name="text">
         <string>Start period</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QDateTimeEdit" name="p_dateTimeEdit_StartPeriod"/>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="p_lbl_EndPeriod">
        <property name="text">
         <string>End period</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QDateTimeEdit" name="p_dateTimeEdit_EndPeriod">
        <property name="enabled">
         <bool>false</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="p_groupBox_WhatWillBeUsed">
     <property name="title">
      <string>Choose what will be used from the existing project</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QCheckBox" name="p_checkBox_UsePositions">
        <property name="text">
         <string>Use a positions</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="p_checkBox_UseWorkers">
        <property name="text">
         <string>Use a workers</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="p_checkBox_UseRequirements">
        <property name="text">
         <string>Use a requirements</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="p_checkBox_AdaptingRequirements">
        <property name="text">
         <string>Adapt requirements to the new project over time</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="p_checkBox_UseWishes">
        <property name="text">
         <string>Use a wishes</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="p_checkBox_AdaptingWishes">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>Adapt wishes to the new project over time</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="p_checkBox_UseShifts">
        <property name="text">
         <string>Use a shifts</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="p_checkBox_AdaptingShifts">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>Adapt shifts to the new project over time</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="p_checkBox_UseAssigned">
        <property name="text">
         <string>Use an assigned shifts</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="p_checkBox_UseVacations">
        <property name="text">
         <string>Use a vacations</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="p_checkBox_AdaptingVacations">
        <property name="text">
         <string>Adapt vacations to the new project over time</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>CreatingProjectBasedOnExistingDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>CreatingProjectBasedOnExistingDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
